frontends/p4/optimizeExpressions.h

```diff
@@ -0,0 +1,24 @@
+#ifndef FRONTENDS_P4_OPTIMIZEEXPRESSIONS_H_
+#define FRONTENDS_P4_OPTIMIZEEXPRESSIONS_H_
+
+#include "frontends/common/constantFolding.h"
+#include "frontends/p4/strengthReduction.h"
+
+namespace IR {
```

Comment:
Headers in frontends/p4 usually put names into `namespace P4`, should we do the same here?